src/views/blocks/tickets/extra-available.php

```diff
@@ -44,30 +42,46 @@
 /**
  * Overwrites the threshold to display "# tickets left".
  *
- * @param int   $threshold Stock threshold to trigger display of "# tickets left"
+ * @since 4.11.1
+ *
  * @param array $data      Ticket data.
  * @param int   $post_id   WP_Post/Event ID.
  *
- * @since 4.11.1
+ * @param int   $threshold Stock threshold to trigger display of "# tickets left"
  */
 $threshold = absint( apply_filters( 'tribe_display_tickets_block_tickets_left_threshold', $threshold, $post_id ) );
-$available = $ticket->available();
 
 /**
  * Allows hiding of "unlimited" to be toggled on/off conditionally.
  *
- * @param int   $show_unlimited allow showing of "unlimited".
- *
  * @since 4.11.1
+ *
+ * @param int $show_unlimited allow showing of "unlimited".
  */
 $show_unlimited = apply_filters( 'tribe_tickets_block_show_unlimited_availability', true, $available );
 ?>
 <div
 	class="tribe-common-b3 tribe-tickets__item__extra__available"
 >
-	<?php if ( $show_unlimited && -1 === $available ) : ?>
-		<?php $this->template( 'blocks/tickets/extra-available-unlimited', array( 'ticket' => $ticket, 'key' => $key ) ); ?>
+	<?php if ( $show_unlimited && - 1 === $available ) : ?>
+		<?php
+		$this->template(
+			'blocks/tickets/extra-available-unlimited',
+			[
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 4713cf51c6680e155f478d304e4e1057</sub>